### PR TITLE
HW-264: Styling jQuery UI Dialog

### PR DESCRIPTION
**Pull request by meladawy**
1- Styling jQuery Dialog UI 
2- Update the markup of Test page to match the target markup
3- Change the label of Dialog to "Preview/ Test"

Before:
before_1
before_2

After:
after_1
after_2

### CODE REVIEW

File: ang/crmMosaico/BlockPreview.html [hunk 1]
```diff
@@ -1,60 +1,53 @@
...
-      class="form-control margin-bottom-10"
-      ng-model="testContact.email"
-      placeholder="example@example.org" />
-
-      <!--fa-paper-plane-->
-      <button class="btn btn-sm btn-primary" title="{{crmMailing.$invalid || !testContact.email ? ts('Complete all required fields first') : ts('Send test message to %1', {1: testContact.email})}}" ng-disabled="crmMailing.$invalid || !testContact.email" ng-click="doSend({email: testContact.email})">{{ts('Send test')}}</button>
+  </div>
+  <div ng-hide="!mailing.body_html && !mailing.body_text" style="margin-top: 1em;" class="btn-group btn-group-justified">
+    <div class="btn-group">
+      <!--<a class="btn btn-sm btn-primary"  ng-disabled="crmMailing.$invalid" ng-click="doPreview('text')">{{ts('Preview as Plain Text')}}</a>  -->
```
Comment (bmnepali) [inline ang/crmMosaico/BlockPreview.html]: Why keep this commented markup?

File: ang/crmMosaico/BlockPreview.html [hunk 2]
```diff
@@ -1,60 +1,53 @@
...
-      ng-options="group.id as group.title for group in crmMailingConst.groupNames|orderBy:'title'"
-      class="form-control margin-bottom-10 full-width-force">
-        <option value=""></option>
-      </select>
+  <input
+  name="preview_test_email"
+  type="email"
+  class="form-control margin-bottom-10"
+  ng-model="testContact.email"
+  placeholder="example@example.org" />
```
Comment (bmnepali) [inline ang/crmMosaico/BlockPreview.html]: Please maintain 2 spaced indentation as follows:
```
 <input
   name="preview_test_email"
   type="email"
   ... />
```

File: ang/crmMosaico/BlockPreview.html [hunk 3]
```diff
@@ -1,60 +1,53 @@
...
-          on-yes="doSend({gid: testGroup.gid})">{{ts('Send test')}}</button>
-    </div>
+  <!--fa-paper-plane-->
+  <button class="btn btn-sm btn-primary" title="{{crmMailing.$invalid || !testContact.email ? ts('Complete all required fields first') : ts('Send test message to %1', {1: testContact.email})}}" ng-disabled="crmMailing.$invalid || !testContact.email" ng-click="doSend({email: testContact.email})">{{ts('Send test')}}</button>
+</div>
 
-  </div>
+<div class="form-group">
+  <label for="preview_test_group">{{ts('Send test email to group:')}} <a crm-ui-help="hs({id: 'test', title: ts('Test Email')})"></a> </label>
+
```
Comment (bmnepali) [inline ang/crmMosaico/BlockPreview.html]: No need of this empty line.

File: ang/crmMosaico/BlockPreview.html [hunk 4]
```diff
@@ -1,60 +1,53 @@
...
+  <select
+  name="preview_test_group"
+  ui-jq="crmSelect2"
+  ui-options="{dropdownAutoWidth : true, allowClear: true, placeholder: ts('Select Group')}"
+  ng-model="testGroup.gid"
+  ng-options="group.id as group.title for group in crmMailingConst.groupNames|orderBy:'title'"
+  class="form-control margin-bottom-10 full-width-force">
```
Comment (bmnepali) [inline ang/crmMosaico/BlockPreview.html]: Adjust the indentation as indicated ^ (2 spaced indentation).

File: ang/crmMosaico/BlockPreview.html [hunk 5]
```diff
@@ -1,60 +1,53 @@
...
+    <option value=""></option>
+  </select>
+
+  <!--fa-paper-plane-->
+  <button class="btn btn-sm btn-primary" title="{{crmMailing.$invalid || !testGroup.gid ? ts('Complete all required fields first') : ts('Send test message to group')}}" ng-disabled="crmMailing.$invalid || !testGroup.gid" crm-confirm="{resizable: true, width: '40%', height: '40%', open: previewTestGroup}"
+      on-yes="doSend({gid: testGroup.gid})">{{ts('Send test')}}</button>
```
Comment (bmnepali) [inline ang/crmMosaico/BlockPreview.html]: Maintain 2 spaced indentation.

**Review by bmnepali — CHANGES_REQUESTED** [review]

File: ang/crmMosaico/BlockPreview.html (hunk 1 above)
Comment (meladawy) [reply ang/crmMosaico/BlockPreview.html]: Actually i didn't change this. i only removed the first 3 lines 

screen shot 2017-04-04 at 5 52 44 pm

But i'll update this in my next commit

**Review by bmnepali — APPROVED** [review]

**Pull request merged**